Qast/Managers/StreamManager.swift

```diff
@@ -57,6 +60,33 @@ class StreamManager {
         return AVPlayerItem(asset: asset)
     }
     
+    public func fetchCoverArtUrl(for isrc: String, _ completion: @escaping (Result<URL>) -> Void) {
```

Comment:
Do you think we should move this out into some other class or manager? I feel like all of the Alamofire logic could be abstracted away. Maybe creating something like a StreamAPI protocol with a function called `fetchCoverArt()` and then making a manager that will handle all of the networking logic

Reply:
definitely doesn't belong in StreamManager. Belongs in something like CapitolAPIRepo or CapitolAPIManager

Will move when we add Title Name functionality, as (I think) that's another call to Capitol